pom.xml

```diff
@@ -1124,33 +1125,53 @@
                         <groupId>org.slf4j</groupId>
                         <artifactId>slf4j-log4j12</artifactId>
                     </exclusion>
```

Comment:
Can we use a wildcard exclusion here?

Reply:
@kokosing either way works: I think the changes in 2.2.8 will help our use case and we were able to deploy 2.2.7 (this pr) recently and it worked as good or better than previous version we were using (2.2.2). Either way I will be testing 2.2.8 in the next day or so - lmk what works best for you. If you want to use the other pr I can test that as well.

Reply:
Updated. I will remove the duplicate finder and paste the results as well and update after we deploy it. And if you want to use the other pr I will test it also.